plugins/dbviewer/api/api.js

```diff
@@ -11,6 +11,24 @@ var common = require('../../../api/utils/common.js'),
         var dbs = {countly: common.db, countly_drill: common.drillDb, countly_out: common.outDb, countly_fs: countlyFs.gridfs.getHandler()};
         var params = ob.params;
         var dbNameOnParam = params.qstring.dbs || params.qstring.db;
+
+        /**
+        * Check properties and manipulate values
+        * @param {object} doc - document
+        * @returns {object} - returns manipulated document object
+        **/
+        function objectIdCheck(doc) {
+            if (typeof doc === "string") {
+                doc = JSON.parse(doc);
+            }
+            for (var key in doc) {
+                if (doc[key] && typeof doc[key].toHexString !== "undefined") {
```

Comment:
let's check === "function" just to make sure there is no case when {"toHexStrin":true} is stored in database :D